boards/arm/stm32f469i_disco/stm32f469i_disco.dts

```diff
@@ -5,7 +5,7 @@
  */
 
 /dts-v1/;
-#include <st/stm32f469.dtsi>
+#include <st/stm32f469iX.dtsi>
```

Comment:
Is that really required to introduce stm32f469iX here, as 6 SPI seems to be a majority on F469
469aX, 469bX, 469iX, 469nX (12 SoCs) : 6 SPI
469vX, 469zX (5 SoCs): 4 SPI

Reply:
deepends on the guidelines we want to introduce. 

I think for 429 (where we now introduced the /delete-node/ example) it was a quite clear case:
 1 (V) vs 5 (ABINZ) series, 5 vs 23 SoCs only has 4 SPIs, others have 6

But for 469 it's not that clear:
 2 (VZ) vs 4(ABIN) series, 5 vs 12 SoCs

Where should we draw this line? Having to delete nodes when introducing new SoCs seems like something that should be avoided where possible.

Reply:
> Where should we draw this line? 

Agreed we're in a grey zone where the line moves depending on the criteria in use.
My point (/criteria) is to limit the number of .dtsi files we'll have to create (and hence maintain).
Then:
-if we introduce a iX.dtsi, we'll need aX, bX and nX : 5 stm32f469 dtsi variants
-if we keep 6 SPI variants in f469.dtsi, we'll required xV and xZ : 3 stm32f469 dtsi variants
I hope this rule is objective enough to be applied easily

Reply:
OK, I'd then propose doing it as in stm32f429.dtsi:

- Introduce the nodes, but put a comment next to them, that lists where exceptions are necessary.
- Whoever introduces a new SoC needs to check the include hierarchy and see if any of the file they include requires deleting a node further down the hierarchy

That should of course be noted in #6199

OK? I'll go ahead and change it in this PR then

Reply:
This would be fine, thanks.